demos/link-prediction/random-walks/cora-lp-demo.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>We've tried to have these buttons end up under the title e.g. https://github.com/stellargraph/stellargraph/blob/develop/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb</p><p> </p><p>This is done by creating a markdown cell that only contains the title, and having any other introductory text be a separate cell after that.</p><p> </p><p>Also, notebook could do with a better title, e.g. "Link prediction with node2vec on Cora" or something.</p><p> </p><p>(I know it's not directly related to the changes in this PR, but I think this PR is enough of a "rewrite the demo" that this can be done here.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I've updated the title and made it its own cell 👍